deploy/nginx/myb.lt

```diff
@@ -0,0 +1,25 @@
+upstream app_myblt {
+    server 127.0.0.1:5000;
+    keepalive 16;
+}
+
+server {
+    root /opt/myblt/public;
+    index index.html;
+
+    server_name myb.lt www.myb.lt;
+
+    #rewrite ^/( ... )$ / last;
```

Comment:
I commented it out because we might want to use it later. If you think we won't need it, I'll gladly remove it.